app/controllers/reports/regions_controller.rb

```diff
@@ -86,16 +86,20 @@ def download
       raise ArgumentError, "Invalid Period #{@period} #{@period.inspect}"
     end
 
-    @cohort_analytics = @region.cohort_analytics(period: @period.type, prev_periods: 6)
     @dashboard_analytics = @region.dashboard_analytics(period: @period.type, prev_periods: 6)
 
     respond_to do |format|
       format.csv do
-        if @region.district_region?
+        if params[:type] == "who_report"
```

Comment:
The fact that I needed to add a new parameter to the payload may indicate that we should just start a new endpoint for this report.

Reply:
Yeah, this should definitely be its own endpoint. I would suggest its own controller entirely, given this is focused on a making CSV downloads for the WHO.